src/lib/update-alternatives/model/alternative.rb

```diff
@@ -0,0 +1,54 @@
+require "cheetah"
+
+module UpdateAlternatives
+  # Represents an alternative
+  class Alternative
+    attr_reader :name
+    attr_reader :status
+    attr_reader :value
+    attr_reader :choices
+    Choice = Struct.new(:path, :priority, :slaves)
```

Comment:
Rubocop does not like `class Alternative < Struct.new(:name, :status, :value, :choices)`
I think it could be this way 
```
  class AlternativeLoader
    Alternative = Struct.new(:name, :status, :value, :choices)
    ...
  end
```